packages/expo-cli/scripts/preversion.js

```diff
@@ -29,7 +29,7 @@ ${bold('3) Eject')}: Create an app and eject it immediately. Check that it build
 
     # Test that it builds for Android
     cd ../android
-    ./gradlew installDevMinSdkDevKernelDebug`
+    ./run.sh`
```

Comment:
@esamelson I got the command from https://docs.expo.io/versions/v31.0.0/expokit/expokit#4-android-build-and-run, should that page in docs also be updated to use `run.sh`?

Reply:
ah yes -- thanks for pointing that out!